user-service/test/test.js

```diff
@@ -0,0 +1,232 @@
+import chai from 'chai'
```

Comment:
Just an fyi, if running npm test locally via CLI, make sure it's not connecting to PROD DB.

Reply:
Noted